src/app/CommandHandler.cpp

```diff
@@ -108,10 +110,15 @@ CHIP_ERROR CommandHandler::ProcessCommandDataElement(CommandDataElement::Parser
 exit:
     if (err != CHIP_NO_ERROR)
     {
+        chip::app::CommandPathParams returnStatusParam = { endpointId,
+                                                           0, // GroupId
+                                                           clusterId, commandId, (chip::app::CommandPathFlags::kEndpointIdValid) };
+
         // The Path is not present when there is an error to be conveyed back. ResponseCommandElement would only have status code,
         // set the error with CHIP_NO_ERROR, then continue to process rest of commands
```

Comment:
This comment no longer looks like it matches the code, not least because a path now _is_ present.